runtime/wasm/runtime.js

```diff
@@ -570,11 +570,22 @@
       "wasm:js-string": string_ops,
       "wasm:text-decoder": string_ops,
       "wasm:text-encoder": string_ops,
+      "": new globalThis.Proxy(
```

Comment:
Do we really need this ? Is it a fallback when the extension is not implemented ?

Reply:
Right. And I think it is still necessary for the version of `node` that we are using on the CI.